tests/test_parse.py

```diff
@@ -124,4 +136,14 @@ def test_method_params():
     # instance default void '.ctor' ()  cil managed
     assert len(methods[1].ParamList) == 0
 
-    methods[0].ParamList[0].row.Name == "args"
+    assert methods[0].ParamList[0].row is not None
+    assert methods[0].ParamList[0].row.Name == "args"
+
+
+def test_ignore_NumberOfRvaAndSizes():
+    # .NET loaders ignores NumberOfRvaAndSizes, so attempt to parse anyways
+    path = fixtures.DATA / "1d41308bf4148b4c138f9307abc696a6e4c05a5a89ddeb8926317685abb1c241"
```

Comment:
Looks like a malware test file / hash made its way into the tests.  I am ok with having malware examples, but not distributing in testfiles, so may make those tests optional for people who do not have the data

Reply:
ah, ok, that's a fair policy. i'll update the test to check for file existence, first

Reply:
fixed in https://github.com/malwarefrank/dnfile/pull/17/commits/877b4caca913537985f4e8dd5ea8805b93350fa9